source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRActionSetRenderer.mm

```diff
@@ -131,41 +127,48 @@ - (UIView *)renderButtons:(ACRView *)rootView
 
         ACOBaseActionElement *acoElem = [[ACOBaseActionElement alloc] initWithBaseActionElement:elem];
 
-        NSUInteger numElem = [[childview arrangedSubviews] count];
-
-        UIButton *button = nil;
+        [self renderButtonForElem:acoElem
+                   actionRenderer:actionRenderer
+                        childview:childview
+                       featureReg:featureReg
+                         rootView:rootView
+                           inputs:inputs
+                        superview:superview
+                       hostConfig:config
+                 accumulatedWidth:accumulatedWidth
+                accumulatedHeight:accumulatedHeight
+                         maxWidth:maxWidth
+                        maxHeight:maxHeight];
+    }
 
-        @try {
-            if ([acoElem meetsRequirements:featureReg] == NO) {
-                @throw [ACOFallbackException fallbackException];
-            }
-            button = [actionRenderer renderButton:rootView
-                                           inputs:inputs
-                                        superview:superview
-                                baseActionElement:acoElem
-                                       hostConfig:config];
-            [childview addArrangedSubview:button];
-        } @catch (ACOFallbackException *exception) {
-            handleActionFallbackException(exception, superview, rootView, inputs, acoElem, config,
-                                          childview);
-            NSUInteger count = [childview.arrangedSubviews count];
-            if (count > numElem) {
-                UIView *view = [childview.arrangedSubviews lastObject];
-                if (view && [view isKindOfClass:[UIButton class]]) {
-                    button = (UIButton *)view;
-                }
-            }
-        }
+    if (!secondary.empty()) {
+        ++renderedBtnNum;
 
-        accumulatedWidth += [button intrinsicContentSize].width;
-        accumulatedHeight += [button intrinsicContentSize].height;
-        maxWidth = MAX(maxWidth, [button intrinsicContentSize].width);
-        maxHeight = MAX(maxHeight, [button intrinsicContentSize].height);
+        ACRBaseActionElementRenderer *actionRenderer =
+            [reg getActionRenderer:[NSNumber numberWithInt:(int)ActionType::Overflow]];
```

Comment:
>[NSNumber numberWithInt:(int)ActionType::Overflow] [](start = 35, length = 50)

can you change it to [ACOBaseActionElement getKey:ActionType::Overflow]?

Reply:
done